cli/cmd/upgrade.go

```diff
@@ -200,7 +200,10 @@ func upgradeRunE(options *upgradeOptions, stage string, flags *pflag.FlagSet) er
 		fmt.Fprintf(os.Stderr, "%s\n", controlPlaneMessage)
 	}
 
-	fmt.Fprintf(os.Stderr, "\n")
+	// check if buffer is not empty
+	if buf.Len() != 0 {
+		fmt.Fprintf(os.Stderr, "\n")
+	}
```

Comment:
This checks that if the output intended for stdout (held in `buf`) is not empty, then append a newline into stderr. What you wanna check instead is if there were messages sent to stderr, then append a new line to stderr.
Also, Github is still complaining about a merge conflict.